src/git/SourceControlPanel.cpp

```diff
@@ -289,20 +291,21 @@ SourceControlPanel::MessageReceived(BMessage *message)
 					{
 						LogInfo("MSG_NOTIFY_PROJECT_SET_ACTIVE");
 						fSelectedProject = gMainWindow->GetActiveProject();
-						_UpdateProjectList();
+						if (!fProjectList->IsEmpty())
+							_UpdateBranchList();
```

Comment:
Is this change intended ? _UpdateProjectList() vs _UpdateBranchList()

Reply:
It's an awful typo, sorry about that